mico-core/src/main/java/io/github/ust/mico/core/broker/MicoApplicationBroker.java

```diff
@@ -400,6 +403,96 @@ public void removeKafkaFaasConnectorInstanceFromMicoApplicationByInstanceId(
         // TODO: Update Kubernetes deployment (see issue mico#627)
     }
 
+    /**
+     * Removes KafkaFaasConnector instances from the {@code kafkaFaasConnectorDeploymentInfos} of the {@link MicoApplication}
+     * that are used in the requested version.
```

Comment:
Removes all ... that have the requested version. Would be easier to understand (if that is the actual intention of the method)